src/test/resources/englishTest.conf

```diff
@@ -2,15 +2,15 @@ include required("eidos.conf") // eidos.conf defaults to English, so start there
 
 EidosSystem {
   // One might turn off geonorm and timenorm, for example.
-  finders = ["gazetteer", "rulebased", "geonorm", "timenorm", "context", "causal", "seasons"]
+  finders = [] // "gazetteer", "rulebased", "geonorm", "timenorm", "context", "causal", "seasons"]
 }
 
 ontologies {
   // These are values that typically must be changed during testing.
-  useGrounding = false
+  useGrounding = true
   //wordToVecPath = ${EidosSystem.path}/w2v/glove.840B.300d.txt // Local resource
   wordToVecPath = /org/clulab/glove/glove.840B.300d.txt // Remote resource
-  ontologies = ["wm_compositional", "un", "wdi", "fao", "mitre12", "who"] // , "interventions", "icasa", "mesh"]
+  ontologies = ["wm_posneg"] // , "interventions", "icasa", "mesh"]
```

Comment:
We should continue to point to wm_compositional by default here.

Reply:
FWIW this file is englishTest.conf used only in testing, not eidos.conf used in production.  Usually testing of grounding is turned off because it is too expensive.